packages/orchestration/test/exos/local-orchestration-account-kit.test.ts

```diff
@@ -304,28 +304,45 @@ test(expectUnhandled(1), 'transfer', async t => {
     ),
   );
 
-  t.like(lastestTxMsg(), {
-    receiver: PFM_RECEIVER,
-    memo: '{"forward":{"receiver":"dydx1test","port":"transfer","channel":"channel-33","retries":3,"timeout":"10m"}}',
+  t.is(lastestTxMsg().receiver, PFM_RECEIVER, 'defaults to "pfm" receiver');
```

Comment:
Has "lastest" become ingrained in our platform, or can it still be changed to "latest"?

Reply:
Hah! That slipped my eye. Updated